pkg/cmd/template.go

```diff
@@ -20,10 +21,34 @@ type compositionData struct {
 func compileCompositionTemplate(path string, input *compositionData) (*bytes.Buffer, error) {
 	templateDir := filepath.Dir(path)
 
+	// Investigate: https://github.com/Masterminds/sprig
+	// Investigate "missingkey=error"
```

Comment:
I think we might be missing some context here. I wouldn't be too sure how to proceed based on this.

Reply:
moved to follow-ups